mesonbuild/modules/python.py

```diff
@@ -149,32 +162,101 @@ def extension_module_method(self, args: T.List['TYPE_var'], kwargs: 'TYPE_kwargs
 
             kwargs['install_dir'] = self._get_install_dir_impl(False, subdir)
 
+        target_suffix = self.suffix
+
+        limited_api_version = kwargs.pop('limited_api', '')
+        if not isinstance(limited_api_version, str):
+            raise InvalidArguments('"limited_api" argument must be a string.')
+
```

Comment:
hmm. Now we have `limited_api` (which is elsewhere a `bool`), `limited_api_suffix`, and `limited_api_version`. Is there a way to unify all that to just `limited_api` which is a string?